resources/lindera_ipadic_conf.json

```diff
@@ -18,6 +18,9 @@
         "dictionary": {
             "kind": "ipadic"
         },
+        "user_dictionary": {
+            "path": "/home/minoru/user_dict.bin"
```

Comment:
It should not have the specific path, right?

Reply:
Thanks. Fixed.